test/e2e/config/caren.yaml

```diff
@@ -201,7 +202,7 @@ variables:
   # # Name of the Prism Element cluster.
   # NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
   # # Name of the OS image pre-uploaded in PC.
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: nkp-rocky-9.4-release-1.30.5-20240930171619
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: nkp-rocky-9.5-release-1.32.3-20250430150550
```

Comment:
lets start moving these to templates please.

Reply:
Go for it - in a follow up PR if you don't mind?